conda/cli/activate.py

```diff
@@ -62,7 +81,7 @@ def main():
             print(os.environ['PATH'])
             raise
         paths = []
-        sys.stderr.write("discarding %s from PATH\n" % binpath)
+        sys.stderr.write("discarding %s from PATH\n" % pathlist_to_str(binpath))
```

Comment:
I have a working prototype here: https://github.com/conda/conda-recipes/pull/439 (depends on quite recent iterations of menuinst, https://github.com/ContinuumIO/menuinst and conda is tied in there a little bit too: https://github.com/conda/conda/pull/1721/files)

General idea is to hard-code the path to activate.bat into the shortcut.  There could be more than one activate.bat, or always just one (in root).  The latter is the current implementation, I think.

Feedback welcome, of course.